tests/assemblyscript/buildscripts/build.ts

```diff
@@ -20,15 +20,32 @@ async function compileTests() {
     const testSuiteFiles = getPathsInDirectory("testsuite");
     const pendingCompilations = testSuiteFiles
         .filter(filePath => doesFileHaveExtension(filePath, "ts"))
-        .map(filePath => ({ src: filePath, wasm: mapFileExtension(filePath, "ts", "wasm")}))
+        .map(filePath => ({ src: filePath, wasm: mapFileExtension(filePath, "ts", "wasm") }))
         .filter(files => isBinaryOutdated(files.src, files.wasm))
-        .map(files => compileWithAsc(files.src, files.wasm));
-    return Promise.allSettled(pendingCompilations);
+        .map(files => compileWithAsc(files.src, files.wasm)
+            .then(_ => undefined)
+            .catch(error => ({ source: files.src, error })));
+
+    const errors = (await Promise.allSettled(pendingCompilations))
+        .map(p => {
+            if (p.status === 'fulfilled' && p.value !== undefined) {
+                return `Failed to compile ${p.value.source}:\n${p.value.error}`;
+            } else if (p.status === 'rejected') {
+                return `Execution failed:\n${p.reason}`;
+            } else {
+                return undefined;
+            }
+        }).filter(p => p !== undefined).map(p => p as string)
```

Comment:
np. I think this last map doesn't do anything as types are stripped on execution. I think it could be replaced by:
```
const errors = ...
        }).filter(p => p !== undefined) as string[]
```